### DISCUSSION TITLE
How to connect to server (select world is better description)

### DISCUSSION THREAD

**Discussion by Zulgeteb** [ORelio/Minecraft-Console-Client#2410] [General]
I can join a server but it runs many worlds (Skyblock/Prison/Bedwars/Survival/Factions).
I have been googling for several days, so far i tried /joinqueue (worked on Minesaga server but not on Pikanetwork)
I tried /join and /server , but doesnt work either.
Then i saw /inventory command was also an option, tried that too but i only manage to click item (compass) in inventory but have no idea how to select Skyblock in menu/container that appears.
So "/inventory player click 36"  works
But when i try "/inventory container click 12" nothing happens
Same for "/inventory 12 rightclick "
Same for "/inventory player click 36 12 rightclick"
![image](https://user-images.githubusercontent.com/45702220/213766287-0bdb5eb6-bfa1-414d-8bd1-064fcda9d2ca.png)
![image](https://user-images.githubusercontent.com/45702220/213767445-c3f3233d-1faa-4f56-ab9a-e898967ed3a9.png)

The next also doesnt seem to work for me:

[[ChatBot.ScriptScheduler.TaskList]]
Task_Name = "Main Script"
Trigger_On_First_Login = false
Trigger_On_Login = true
Trigger_On_Times = { Enable = false, Times = [ 23:59:59, ] }
Trigger_On_Interval = { Enable = false, MinTime = 1.2, MaxTime = 1.5 }
Action = "script main.txt"

No idea what i am doing wrong.
So i have to select Compass > Grass block > Podzol block

**Comment by BruceChenQAQ** [comment]
Can you provide the server address, I'll test it.

**Reply by Zulgeteb** [reply]
play.pika-network.net

**Comment by BruceChenQAQ** [comment]
You need to use the compass held in your hand via `/useitem` instead of clicking on it in the player's inventory.

main.txt
```
wait 10
changeslot 1
wait 1
useitem
wait 10
inventory container click 12
wait 10
inventory container click 11
```

**Reply by Zulgeteb** [reply]
Yess my friends this works, thanks a lot.
